spikeinterface/sorters/tests/test_runsorter.py

```diff
@@ -18,22 +19,46 @@ def test_run_sorter_local():
     print(sorting)
 
 
-@pytest.mark.skip('Docker test no run with pytest : do it manually')
+@pytest.mark.skip("Docker tests don't run with pytest : test it manually")
 def test_run_sorter_docker():
-    local_path = download_dataset(remote_path='mearec/mearec_test_10s.h5')
-    recording, sorting_true = read_mearec(local_path)
+    mearec_filename = download_dataset(remote_path='mearec/mearec_test_10s.h5')
+
+    base_folder = Path('/data_local/DataSpikeSorting/mearec')
+    mearec_filename = base_folder / 'recordings_collision_10cells_Neuronexus-32_180s.h5'
+    recording, sorting_true = read_mearec(mearec_filename)
 
     sorter_params = {'detect_threshold': 4.9}
 
-    docker_image = 'spikeinterface/tridesclous-base:1.6.3'
+    docker_image = 'spikeinterface/tridesclous-base:1.6.4'
 
-    sorting = run_sorter('tridesclous', recording, output_folder='sorting_tdc_docker',
+    sorting = run_sorter('tridesclous', recording, output_folder=base_folder/'sorting_tdc_docker',
                          remove_existing_folder=True, delete_output_folder=False,
                          verbose=True, raise_error=True, docker_image=docker_image,
                          **sorter_params)
     print(sorting)
 
 
+@pytest.mark.skip("Singularity tests don't run with pytest : test it manually")
+def test_run_sorter_singularity():
+    #mearec_filename = download_dataset(remote_path='mearec/mearec_test_10s.h5')
+
+    base_folder = Path('/mnt/data/sam/DataSpikeSorting/mearec')
+    mearec_filename = base_folder / 'recordings_collision_10cells_Neuronexus-32_180s.h5'
```

Comment:
same here: better use a downloaded files. Then at least the test can be run in the same way locally by different people.